src/main/java/core/basesyntax/model/io/FileReaderImpl.java

```
@@ -0,0 +1,24 @@
+package core.basesyntax.model.io;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+
+    @Override
+    public List<String> read(String filePath) {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader br = new BufferedReader(new java.io.FileReader(filePath))) {
+            String line;
+            br.readLine();
+            while ((line = br.readLine()) != null) {
+                lines.add(line.trim());
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Could not read file: " + filePath, e);
```

Error handling: you already catch IOException and rethrow `new RuntimeException("Could not read file: " + filePath, e);` which matches checklist guidance to rethrow informative RuntimeException with the original cause. Keep this behavior (good). See checklist: "Don't ignore exceptions in try-catch blocks ... rethrow a RuntimeException with an informative message and exception object."